cloudflare-rails.gemspec

```diff
@@ -31,9 +31,9 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "pry-byebug"
   spec.add_development_dependency "appraisal"
 
-  spec.add_dependency "railties", ">= 5.2", "< 6.2.0"
-  spec.add_dependency "activesupport", ">= 5.2", "< 6.2.0"
-  spec.add_dependency "actionpack", ">= 5.2", "< 6.2.0"
+  spec.add_dependency "railties", ">= 5.2", "< 7.1.0"
+  spec.add_dependency "activesupport", ">= 5.2", "< 7.1.0"
+  spec.add_dependency "actionpack", ">= 5.2", "< 7.10"
```

Comment:
Hey @modosc is this expected? (`7.10` instead of `7.1.0`)

Reply:
NOPE fixing now

Reply:
@cover i'm about to merge #47  - do you think i need to pull the `2.1.0` release over this? i can't see how it would break anything but you obviously noticed somehow?

Reply:
i went ahead and yanked it, thanks for the catch

Reply:
I don't think it'd have broken anything, I noticed it while I was inspecting the `Gemfile.lock` diff after upgrading to `2.1.0`.

Maybe even a `2.1.1` with no yanking was fine. Thank you for fixing it! :slightly_smiling_face: